pipeline/e2e-test-linux-docker.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+steps:
+    - script: docker build -t app .
+      displayName: setup docker
+
+    - script: docker run --network=host -i app
+      displayName: run e2e tests on docker
+
+    - bash: |
```

Comment:
I hit my timebox on the `-v` parameter, so let's keep this way for the time being